api/default.nix

```diff
@@ -52,6 +58,9 @@ with pkgs; rec {
         echo "nixbld$i:x:$(( $i + 30000 )):30000:Nix build user $i:/var/empty:/run/current-system/sw/bin/nologin" >> etc/passwd
       done
 
+      mkdir -p usr/bin
```

Comment:
Whats usr/bin needed for?

Reply:
It is needed for nodemon and for the test.js